docs/guides/external-node/07_snapshots_recovery.md

```diff
@@ -0,0 +1,26 @@
+# Snapshots Recovery
+
+Instead of starting node using DB snapshots, it's possible to configure them to start from a protocol-level snapshots.
+This process is much faster and requires way less storage. Postgres database of a mainnet node recovered from a snapshot
+is only about 300GB.
```

Comment:
It's worth mentioning that without pruning enabled, the node storage will continuously grow; probably makes sense to mention the rate of growth as well.

Reply:
+1